carina-api/src/main/java/com/qaprosoft/apitools/builder/GenerateProcessor.java

```diff
@@ -32,36 +32,37 @@ public Properties process(Properties in) {
-                    Matcher tmpMatcher = Pattern.compile("\\d+").matcher(toReplace);
-                    tmpMatcher.find();
-                    String length = tmpMatcher.group();
-                    out.put(entry.getKey(), entry.getValue().toString().replace(toReplace, GenerationUtil.generateNumber(Integer.parseInt(length))));
-                } else if (dateMatcher.find()) {
-                    String toReplace = dateMatcher.group();
-                    // getting offset
-                    Matcher offsetMatcher = Pattern.compile("-{0,1}\\d+").matcher(entry.getValue().toString());
-                    offsetMatcher.find();
-                    String offset = offsetMatcher.group();
-                    // getting format
-                    Matcher formatMatcher = Pattern.compile("(?<=generate_date\\().*(?=;)").matcher(entry.getValue().toString());
-                    formatMatcher.find();
-                    String format = formatMatcher.group();
-                    // generating date
-                    out.put(entry.getKey(), entry.getValue().toString().replace(toReplace,
-                            GenerationUtil.generateTime(format, Integer.parseInt(offset), Calendar.DAY_OF_YEAR)));
-                } else {
-                    out.put(entry.getKey(), entry.getValue());
-                }
+            String tmp = entry.getValue().toString();
+
+            while (wordMatcher.find()) {
+                String toReplace = wordMatcher.group();
+                Matcher tmpMatcher = Pattern.compile("\\d+").matcher(toReplace);
+                tmpMatcher.find();
+                String length = tmpMatcher.group();
+                tmp = tmp.replace(toReplace, GenerationUtil.generateWord(Integer.parseInt(length)));
             }
+            while (numberMatcher.find()) {
+                String toReplace = numberMatcher.group();
+                Matcher tmpMatcher = Pattern.compile("\\d+").matcher(toReplace);
+                tmpMatcher.find();
+                String length = tmpMatcher.group();
+                tmp = tmp.replace(toReplace, GenerationUtil.generateNumber(Integer.parseInt(length)));
+            }
+            while (dateMatcher.find()) {
+                String toReplace = dateMatcher.group();
+                // getting offset
+                Matcher offsetMatcher = Pattern.compile("-{0,1}\\d+").matcher(toReplace);
+                offsetMatcher.find();
+                String offset = offsetMatcher.group();
+                // getting format
+                Matcher formatMatcher = Pattern.compile("(?<=generate_date\\().*?(?=;)").matcher(toReplace);
+                formatMatcher.find();
+                String format = formatMatcher.group();
+                // generating date
+                tmp = tmp.replace(toReplace,
+                        GenerationUtil.generateTime(format, Integer.parseInt(offset), Calendar.DAY_OF_YEAR));
+                }
```

Comment:
@KirillBugrim formatting issue in lines 64-65 which make an illusion of the mistake... 
out.put should be on the same level and while operators

Reply:
@vdelendik fixed